Apps/Examples/Examples/All Examples/LiveDataExample.swift

```diff
@@ -0,0 +1,96 @@
+import MapboxMaps
+import Turf
+
+@objc(LiveDataExample)
+
+class LiveDataExample: UIViewController, ExampleProtocol {
+    var mapView: MapView!
+    let sourceId = "drone-source"
+    var droneTimer: Timer?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let mapInitOptions = MapInitOptions(cameraOptions: CameraOptions(zoom: 0))
+        mapView = MapView(frame: view.bounds, mapInitOptions: mapInitOptions)
+        mapView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+
+        view.addSubview(mapView)
+
+        // Add the live data layer once the map has finished loading.
+        mapView.mapboxMap.onNext(.mapLoaded) { _ in
+            self.addStyleLayer()
+        }
+    }
+
+    func addStyleLayer() {
+        // Set the data for the `GeoJSONSource`. This URL simulates paths
+        // for drones.
+        guard let url = URL(string: "https://wanderdrone.appspot.com/") else { return }
```

Comment:
No need to guard here. We know that the string we're passing to `URL(string:)` has the structure of a valid URL, so we can safely force unwrap.

```suggestion
        let url = URL(string: "https://wanderdrone.appspot.com/")!
```